css-canvas-medical/index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Canvas Medical</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="indented border-bottom">
+      <div class="row">
+        <img class="logo" src="./images/logo-blue.svg" alt="canvas logo" />
+        <div class="column-one-half">
+          <div class="row">
+            <a class="header-link-1 fs15pt" href="./product.html">Product</a>
+            <a class="header-link-1 fs15pt" href="./implementation.html"
+              >Implementation</a
+            >
+            <a class="header-link-1 fs15pt" href="./about-us.html">About Us</a>
+            <a class="header-link-1 fs15pt" href="./blog.html">Blog</a>
+            <a class="header-link-2 blue-button fs15pt white-text"
+              >Contact Us</a
+            >
+          </div>
```

Comment:
Because this is essentially a _list_ of links, I would have put this into a `nav` and `ul` element and each `a` be inside of an `li`. Functionally doing the same thing you have here but semantically a screen reader would note that it is a navigation list with five items and ready out each link.